bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller_test.go

```diff
@@ -1400,6 +1444,7 @@ func TestKubeadmConfigReconciler_ClusterToKubeadmConfigs(t *testing.T) {
 		names[i] = configs[i].Name
 	}
 	for _, name := range expectedNames {
+		log.Print(name)
```

Comment:
Leftover from debugging, I think

Reply:
Just fyi, in case you're not aware and it might be helpful sometimes.

With Intellij you can set non-breaking logging breakpoints. When you run the tests with debug, Intellij then evaluates the statement and logs it to stdout during runtime: (and apparently in another color)

![image](https://user-images.githubusercontent.com/4662360/140027138-7e8ac192-a560-436e-92bc-2cb925b3749d.png)


What makes this really really great is that you can of course do the same in dependencies like the Go SDK or in controller-runtime, where it's impossible or very hard to "just" add log statements.

(@fabriziopandini @ykakarap @randomvariable  maybe also helpful for you)